resources/user.rb

```diff
@@ -72,7 +72,7 @@ def run_query(query)
 
   def database_has_password_column
     begin
-      result = run_query('SHOW COLUMNS FROM mysql.user WHERE Field="Password"')
+      result = run_query("SHOW COLUMNS FROM mysql.user WHERE Field='Password';")
```

Comment:
Why the change here?

Reply:
This setup of quotes is what causes #220, the query is formed to:

```
"SHOW COLUMNS FROM mysql.user WHERE Field="Password"
```

which results in a non-valid query